apps/meteor/app/apps/server/bridges/listeners.js

```diff
@@ -85,7 +85,7 @@ export class AppListenerBridge {
 						message: msg,
 						user: this.orch.getConverters().get('users').convertToApp(userReacted),
 						reaction,
-						isRemoved,
+						isReacted: isRemoved,
```

Comment:
I suggest changing the name of the variable being destructured in the parameters array, to `isReacted` to maintain the code cleaner

Reply:
Thanks @tapiarafael. will make the required changes.

Reply:
Hey @tapiarafael, made the neccessary changes as per suggestion you gave.